source/CardGames.py

```diff
@@ -403,3 +403,12 @@ def playerTurn(players: list, current_player: str, getCard: bool) -> None:
       next_player_index = (players.index(current_player) + 1) % num_players
       next_player = players[next_player_index]
       return next_player
+
+def start_game_window():
+    print("**************************")
+    print("*                        *")
+    print("*  Welcome to Go Fish    *")
+    print("*                        *")
+    print("**************************\n")
+    num_players = input("How many players would you like to have? ")
+    return int(num_players)
```

Comment:
This is a cool print statement, however, I think you can be a bit more creative with it. I also thought we mentioned during our planning that the rules of the game would be added to the start screen, maybe you could add them and explain how the game works.

Reply:
Interesting approach for the startup screen. But now we have an inconsistency with the start and end screens since one uses tkinter for an external display and the other works in the terminal. We should figure out which method we want to commit to and have them be the same to avoid conflicts.

Reply:
Those are both great points guys, and I will take your comments into consideration and see what I can improve about the code.